include/clink/kernels/opdefs/clink_kernels.td

```diff
@@ -76,4 +90,54 @@ def SquareF64Op: Clink_Op<"square.f64"> {
   let verifier = ?;
 }
 
+def OneHotEncoderLoadOp: Clink_Op<"onehotencoder_load"> {
```

Comment:
If we decide to add generic kernels to do load/transform for all model subclasses, it is probably better to avoid checking in the code that we will remove soon.

Reply:
As discussed offline, we plan to provide generic kernels for load/transform in the next PR. Before that PR we may still need such specified kernels to support the computation logic of OneHotEncoder. I can add a TODO here stating that this kernel is supposed to be removed soon in the next PR. How do you like this idea?

Reply:
Sounds good. Feel free to use a TODO for now.